src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  font-family: Roboto, Helvetica, sans-serif;
   margin: 0;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  width: auto;
+  justify-content: space-between;
+  text-align: center;
+  align-items: center;
+}
+
+.logo {
+  align-items: center;
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: inline-flex;
+  height: 60px;
+}
+
+ul {
+  margin: 0;
+}
+
+ul li:last-child {
+  margin-right: 50px;
+}
+
+.nav_item {
+  display: inline-flex;
+  list-style-type: none;
+}
+
+.nav_link {
+  height: 100%;
+  text-decoration: none;
+  margin-left: 20px;
+  line-height: 60px;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```

Comment:
CSS Variable is used for a blue color X
<img width="498" alt="image" src="https://github.com/user-attachments/assets/7f3b919e-b36f-4dfe-8ab7-a3836ff2757e">